# Keypoint2D.msg
int32 class_id
float32 x
float32 y
